Javascript/README.md

```diff
@@ -0,0 +1,92 @@
+# Bitcamp JavaScript :rocket: 
+
+JavaScript is an extremely useful tool for user interface (UI) design that allows the programmer to build interactive webpages.  Some of the most essential items for a website's success are ease of navigation and an appealing design, thus making mastery of JavaScript a must for aspiring designers.
+
+## The Goal 🥅
+
+**Who?**  The material in this program is simplified so that those with minimal experience in computer science can try their hand at web design. This bootcamp was mainly designed for high school and college students, but aspiring coders of all ages are welcomed. Our instructors have an extensive background in coding, but are not experts with JavaScript.
+
+**What?** Concise and interactive Github Learning Lab that introduces and develops skills related to JavaScript with [Twilio](https://www.twilio.com/quest).
+
+**How?** Students will get experience with JavaScript, learning the basics as well as some advanced topics by completing assignments every week. In the end, students will use the knowledge theyve gained from TwilioQuest, a PC roleplaying game, to build a game of their choosing. 
+
+**When?** 
+
+* *Starting out:* Weeks 1 and 2 will help you produce simple JavaScript programs to help you get acclimated with all different aspects of the language.
+* *The project:* In week 3, we will go over more advanced topics like recursion and making graphics in JS. In the fourth and final week, you will use all of your newly-acquired skills to build a game in TwilioQuest.
+
+**Where?** Tools used include Github, JavaScript, Linux/OS Terminal, and TwilioQuest.
+
+### Project Description
+
+4-week course on using Javascript and Twilio.
```

Comment:
I think we should change this to a 6 week course and instead of week 3 being recursion, have weeks 3-4 about html/css/building a bare bones website and integrating js into the website, giving them week 5-6 to work on their own website with javascript? maybe something similar to what Emily Gui's team did for the IMC last year? you can use that project as the base for weeks 3-4